test/features/test_kaldi_features.py

```diff
@@ -77,8 +99,25 @@ def test_kaldi_mfcc_extractor(recording):
 
 def test_kaldi_mfcc_extractor_vs_torchaudio(recording):
     audio = recording.load_audio()
-    fbank = Mfcc()
-    fbank_ta = TorchaudioMfcc()
-    feats = fbank.extract(audio, recording.sampling_rate)
-    feats_ta = fbank_ta.extract(audio, recording.sampling_rate)
+    mfcc = Mfcc()
+    mfcc_ta = TorchaudioMfcc()
+    feats = mfcc.extract(audio, recording.sampling_rate)
+    feats_ta = mfcc_ta.extract(audio, recording.sampling_rate)
     torch.testing.assert_allclose(feats, feats_ta)
+
+
+def test_kaldi_spectrogram_extractor(recording):
+    spec = Spectrogram()
+    feats = spec.extract(recording.load_audio(), recording.sampling_rate)
+    assert feats.shape == (1604, 257)
+
+
+def test_kaldi_spectrogram_extractor_vs_torchaudio(recording):
```

Comment:
This test is currently failing with the following message:
```python
E       AssertionError: Tensor-likes are not close!
E       
E       Mismatched elements: 3874 / 412228 (0.9%)
E       Greatest absolute difference: 22.66506841033697 at index (826, 0) (up to 1e-05 allowed)
E       Greatest relative difference: 17694.401589361405 at index (1029, 0) (up to 0.0001 allowed)
```

Reply:
On changing the assertion to `torch.testing.assert_allclose(feats, np.exp(feats_ta))`, I get:
```python
E       AssertionError: Tensor-likes are not close!
E       
E       Mismatched elements: 1604 / 412228 (0.4%)
E       Greatest absolute difference: 8.267975032700633 at index (325, 0) (up to 1e-05 allowed)
E       Greatest relative difference: 0.999999999856555 at index (826, 0) (up to 0.0001 allowed)
```
1604 is the number of frames, and it seems they only differ in the 0th dimension.

Reply:
Ahh, the first dimension is the log energy for both, so we don't need to exponentiate that for comparing.